pkgs/os-specific/darwin/apple-sdk-12.3/frameworks/default.nix

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
this code looks like a copy / paste of apple-sdk-11.0/apple_sdk.nix

is there any way to unify them a little?